examples/04_simplest.html

```diff
@@ -24,14 +24,13 @@ <h2>04 _ simplest possible example</h2>
 				var output = document.createElement( 'div' );
 				output.style.cssText = 'position: absolute; left: 50px; top: 300px; font-size: 100px';
 				document.body.appendChild( output );
-
+				var startValue = { x: 50, y: 0 };
```

Comment:
@BoTime Thanks for making this effort. Have you tested this example? Looks like you need to pass startValue into the new Tween, otherwise the value output is constant.